### ISSUE TITLE
Unable to install on Rails 7 app

### ISSUE THREAD

**Issue by Shpigford** [SpinaCMS/Spina#918]
Gemfile: `gem 'spina', git: 'https://github.com/SpinaCMS/Spina'`

```
Bundler could not find compatible versions for gem "activerecord":
  In snapshot (Gemfile.lock):
    activerecord (= 7.0.0)

  In Gemfile:
    spina was resolved to 2.6.1, which depends on
      attr_json was resolved to 0.5.0, which depends on
        activerecord (< 5.3, >= 5.0.0)

    rails (~> 7.0.0) was resolved to 7.0.0, which depends on
      activerecord (= 7.0.0)
```

Rails 7 app running Ruby 3.0.2

**Comment by Bramjetten** [comment]
Hey @Shpigford great to see you here! 

I believe attr_json works fine on Rails 7 and Ruby 3. Their gemspec still requires < 7.0.0 though. I’ll ask them to release a new gem so Spina can properly support Rails 7 as well!

You might want to check out @peterberkenbosch’s PR, he’s already running Rails 7 I believe.

**Comment by Bramjetten** [comment]
I asked the maintainer to release a new gem for Rails 7: https://github.com/jrochkind/attr_json/issues/134

**Comment by Bramjetten** [comment]
`attr_json` was just updated to support Rails 7. I also released a new version (`v2.6.2`) of Spina which includes the latest gem dependencies. Should work great on Rails 7!
